src/components/DayOverview/DayOverviewPdf.tsx

```diff
@@ -0,0 +1,182 @@
+"use client";
+
+import React from "react";
+import { Svg, Document, Page, Text, View, StyleSheet, Path } from "@react-pdf/renderer";
+import { faFlag, faSquare, IconDefinition } from "@fortawesome/free-solid-svg-icons";
+
+import { ParcelOfSpecificDateLocation } from "@/components/DayOverview/DayOverview";
+interface DayOverviewPdfProps {
+    date: Date;
+    location: string;
+    data: ParcelOfSpecificDateLocation[];
+}
+
+interface DayOverviewCardProps extends DayOverviewPdfProps {}
+
+interface CustomSVGProps {
+    icon: IconDefinition;
+    color: string;
+    fill: boolean;
+}
+
+const styles = StyleSheet.create({
+    page: {
+        display: "flex",
+        flexDirection: "column",
+        fontSize: "12px",
+    },
+    card: {
+        margin: "0px 40px 20px",
+    },
+    table: {
+        borderStyle: "solid",
+        borderWidth: 1,
+        borderRightWidth: 0,
+        borderBottomWidth: 0,
+    },
+    title: {
+        fontFamily: "Helvetica-Bold",
+        textTransform: "uppercase",
+        fontSize: "30px",
+        textAlign: "center",
+        margin: "30px",
+    },
+    dateLocationWrap: {
+        justifyContent: "space-between",
+        marginBottom: "5px",
+    },
+    svg: {
+        margin: "0 1px",
+        padding: "0 1px",
+    },
+    cell: {
+        borderStyle: "solid",
+        borderWidth: 1,
+        borderLeftWidth: 0,
+        borderTopWidth: 0,
+        padding: "3px",
+    },
+    row: { display: "flex", flexDirection: "row" },
+    bold: { fontFamily: "Helvetica-Bold" },
+
+    // cells
+    cell_logo: { flex: 1 },
+    cell_name: { flex: 4 },
```

Comment:
A lot of the cell here has the same thing eg. flex: 4, can we combine this?

Reply:
Probably might not, since the number are not technically supposed to have any correlation (it is just coincidentally 4 looks nice for those columns) - so if I change one column's flex number it should not change the other columns one if future developers want to improve the layout

Reply:
I agree I would leave these separate!